GetMethod/GetMethod.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+namespace tvm { namespace schema {
+
+__interface IGetMethod {
+
+	[[internal]]
+	void run(uint32 answerId, cell message);
```

Comment:
c++ doesn't need this interface, getter may be executed just as external message sent from debot. Engine may detect that this is getter call if no pubkey/signature in the message.

Reply:
Show the example, please.